static/src/stylesheets/base/_helpers.scss

```diff
@@ -14,12 +14,6 @@
     }
 }
 
-.hide-until-mobile-medium {
```

Comment:
do we wanna keep these ones around?

Reply:
Nope! 🔥 

🔪 💀 Delete all the CSS you can 👍